webrender/src/renderer.rs

```diff
@@ -2304,7 +2308,7 @@ impl Renderer {
 
             self.device.disable_scissor();
             self.device.disable_depth();
-            self.device.set_blend(false);
+            self.set_blend(false, true);
```

Comment:
uh, that's a bit confusing when reading the code.
can have an `enum TargetKind` or something like this?